.eslintrc.json

```diff
@@ -15,6 +15,17 @@
       "import/no-extraneous-dependencies": [
         "error", { "devDependencies": true }
       ],
+      "no-plusplus": [
+        "error", { "allowForLoopAfterthoughts" : true }
```

Comment:
I opted to allow the no-plusplus rule in moment-opening-times as it seemed better to not have a mix within the code.

What was your rational for going with allowForLoopAfterthoughts?